chore(docs): migrate cloud docs to dotcom(1) (#36452)
gatsbyjs/gatsby: docs/docs/reference/cloud/build-logs.md

---
title: "Build Logs"

> description: "A guide for viewing and understanding build logs on Gatsby Cloud"

Reviewer: ```suggestion
description: "Learn how to view and understand build logs on Gatsby Cloud"
```
